src/AbstractPluginManager.php

```diff
@@ -255,9 +255,7 @@ public function setInvokableClass(string $name, string|null $class = null): void
      *
      * @deprecated Please use {@see AbstractPluginManager::configure()} instead.
      *
-     * @phpcs:disable Generic.Files.LineLength.TooLong
-     * @param class-string<FactoryInterface>|class-string<object&FactoryCallable>|FactoryCallable|FactoryInterface $factory
-     * @phpcs:enable Generic.Files.LineLength.TooLong
+     * @param class-string<FactoryInterface>|FactoryCallable|FactoryInterface $factory
```

Comment:
Using intersections with callable in a class string parameter is not supported, i.e. `class-string<object&callable>` causes an `InvalidDocblock` issue in Psalm